src/Gui/Forms/MainFormInteractor.cs

```diff
@@ -329,11 +329,57 @@ public void AddMetadataFile()
         /// <summary>
         /// Prompts the user for a script file and adds to the project.
         /// </summary>
-        public void AddScriptFile()
+        private void AddScriptFile()
         {
             var fileName = uiSvc.ShowOpenFileDialog(null);
             if (fileName == null)
                 return;
+            AddScriptFile(fileName);
+        }
+
+        /// <summary>
+        /// Prompts the user for a creating new script file and adds it to the
+        /// project.
+        /// </summary>
+        private void NewScriptFile()
+        {
+            var fileName = uiSvc.ShowSaveFileDialog(GetDefaultScriptPath());
+            if (fileName == null)
+                return;
+            var fsSvc = sc.RequireService<IFileSystemService>();
+            try
+            {
+                fsSvc.CopyFile(GetScriptTemplatePath(), fileName, true);
+                AddScriptFile(fileName);
+            }
+            catch (Exception e)
+            {
+                uiSvc.ShowError(
+                    e,
+                    "An error occured while creating the script file {0}.",
+                    fileName);
+            }
+        }
+
+        private string GetDefaultScriptPath()
+        {
+            var defaultFileName = "new_script.py";
+            return ProjectPersister.ConvertToAbsolutePath(
+                ProjectFileName, defaultFileName);
+        }
+
+        private string GetScriptTemplatePath()
+        {
+            var cfgSvc = sc.RequireService<IConfigurationService>();
+            var templName = "_new_script_template.py";
```

Comment:
I think it is time to start organizing the Reko output directory. We are dumping all files in the same place and it's becoming disorganized. I suggest creatiing a subfolder in the output directory called `templates` and putting this file there.

Reply:
Would not be better to copy all `Python` files (`_reko.py`, `_program.py`, `_new_script_template.py`) to 
same directory (e.g. create `Python` folder)?

Reply:
I have moved `*.py` files to `Python` directory in 11ffb8e